langstream-api/src/main/java/ai/langstream/api/runner/code/AgentContext.java

```diff
@@ -36,6 +36,19 @@ public interface AgentContext {
 
     TopicProducer getTopicProducer();
 
+    /**
+     * Unique id of the agent relative to the application.
+     *
+     * @return
+     */
+    String getAgentId();
+
+    /**
+     * Unique id of the agent relative to the tenant. This is normally computed by
+     * {applicationId}-{agentId}
```

Comment:
this is basically the id of the "executor", that is of the statefulset.
it is not about the "agent"

This comment is wrong